hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestContainerBalancerOperations.java

```diff
@@ -115,4 +125,47 @@ public void testContainerBalancerCLIOperations() throws Exception {
   }
 
   //TODO: add more acceptance after container balancer is fully completed
+
+  /**
+   * Test if Container Balancer CLI options override the default configurations.
+   */
```

Comment:
Let's also test that a CLI option overrides an option specified in the configs. For example, if balancing interval is 40 in ozone-site.xml but 30 is passed in through the CLI, 30 should be used.